packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -61,7 +61,9 @@ export const upgradeVaults = async (powers, { options }) => {
     instance: {
       produce: { auctioneer: auctioneerProducer },
     },
-  } = powers;
+  },
+  { options },
+) => {
```

Comment:
I'm struggling to understand this change. Is it just formatting?

Reply:
It's mostly formatting. The old code had access to the name `powers` within the body. That was used in some proposals in order to pass all the powers to sub-proposals. That isn't happening here, so I reverted to the more conventional (for proposals) format of expanding `powers` in the parameter list, rather than passing it through and expanding it in the body.

It was just (bad?) luck that the differ matches the expansion of powers in the param list with the identical one in the body.

I would have accepted a suggestion that it was unrelated to this PR, but now that I'm changing the expansion of powers in order to get `installation.produce.VaultFactory`, I'll leave it in as cleanup related to that.